pom.xml

```diff
@@ -2161,9 +2182,6 @@
           <shadedArtifactAttached>false</shadedArtifactAttached>
           <artifactSet>
             <includes>
-              <!-- At a minimum we must include this to force effective pom generation -->
-              <include>org.spark-project.spark:unused</include>
-
               <include>org.eclipse.jetty:jetty-io</include>
```

Comment:
No, we're still shading Guava and Jetty. The output of a maven build should not contain any jetty jars (I think I checked that but can double check it).